main.qml

```diff
@@ -1891,6 +1891,17 @@ ApplicationWindow {
         passwordDialog.open();
     }
 
+    function getDefaultDaemonRpcPort(networkType) {
```

Comment:
Probably best to put this function in `js/Utils.js` and use it via `Utils.getDefaultDaemonRpcPort()`. You will need `import "../js/Utils.js" as Utils`